components/LocaleSwitcher.vue

```diff
@@ -5,7 +5,8 @@
         <v-icon>mdi-web</v-icon>
       </v-btn>
     </template>
-    <v-list>
+    <v-list style="max-height: 90vh"
```

Comment:
I'd rather have that in a scoped style block at the end of the file, if possible.

90vh is also probably too high. Something in EM would be better, for consistency.